browser/profiles/brave_profile_manager.cc

```diff
@@ -82,10 +87,20 @@ void BraveProfileManager::InitTorProfileUserPrefs(Profile* profile) {
 #endif
 }
 
+// static
+void BraveProfileManager::InitGCMPrefs(Profile* profile) {
```

Comment:
why are these here instead of setting the correct values in RegisterProfilePrefs?

Reply:
I'm not sure if there is a way to access a pref value (`gcm::prefs::kGCMChannelStatus`) in RegisterProfilePrefs, checking!